server/routes/index.js

```diff
@@ -5,7 +5,7 @@ const { createTopicsRouter } = require('./topics');
 const { createTimetableRouter } = require('./timetable');
 const { createContentRouter } = require('./content');
 const { createMoneyRouter } = require('./money');
-const { createApprovedVisitorsRouter } = require('./approvedVisitors');
+// const { createApprovedVisitorsRouter } = require('./approvedVisitors');
```

Comment:
Commented out to prevent access until views updated in an upcoming ticket and testing session has been arranged.